EIPS/eip-2612.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 2612
+title: permit: 712-signed approvals
```

Comment:
I *suspect* EIP-2711 is going to go through eventually (my hope is in the HF after Berlin).  You may want to consider whether this EIP is still valuable after that, and if not whether it is worth standardizing if it will only be valuable for a relatively short amount of time.